drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -1887,8 +1887,8 @@ void RasterizerSceneGLES3::_setup_light(RenderList::Element *e, const Transform
 		const RID *lights = e->instance->light_instances.ptr();
 
 		for (int i = 0; i < lc; i++) {
-			LightInstance *li = light_instance_owner.getptr(lights[i]);
-			if (li->last_pass != render_pass) //not visible
+			LightInstance *li = light_instance_owner.getornull(lights[i]);
+			if (li && li->last_pass != render_pass) //not visible
```

Comment:
Should be `if ( !li || li->last_pass` like in GLES 2

Reply:
Good idea. I was using ``li`` to avoid a null pointer read, but ``!li ||`` is much better to avoid issues further down.